app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreen.kt

```diff
@@ -329,6 +349,31 @@ private fun VaultItemListingDialogs(
             )
         }
 
+        is VaultItemListingState.DialogState.Fido2PinSetUpPrompt -> {
+            var pin by rememberSaveable { mutableStateOf("") }
+            PinInputDialog(
```

Comment:
Would it make sense to make a helper function that lives with the existing `PinInputDialog`

```kotlin
/**
 * Documentation!!!
 */
@Composable
fun PinInputDialog(
    onCancelClick: () -> Unit,
    onSubmitClick: (String) -> Unit,
    onDismissRequest: () -> Unit,
) {
    var pin by remember { mutableStateOf("") }
    PinInputDialog(
        pin = pin,
        onPinChange = { pin = it },
        onCancelClick = onCancelClick,
        onSubmitClick = { onSubmitClick(pin) },
        onDismissRequest = onDismissRequest,
    )
}
```

Reply:
That's clever, thanks!